crates/sui-framework/packages/sui-system/tests/rewards_distribution_tests.move

```diff
@@ -628,4 +630,59 @@ module sui_system::rewards_distribution_tests {
         test_scenario::return_shared(sui_system);
         test.end();
     }
+
+    #[test]
+    fun test_process_pending_stake_withdraw_no_underflow_in_safe_mode() {
+        let start_epoch: u64 = 1;
+        let epoch_start_time = 100000000000;
+
+        set_up_sui_system_state();
+        let mut scenario_val = test_scenario::begin(VALIDATOR_ADDR_1); // val 1 has 100 staked sui
```

Comment:
While I know it's named in a similar way in other tests, we should move away from this naming scheme with `_val`. In some tests we use `test` or `scenario`.